device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,146 @@
+components:
+  schemas:
+    Device.Dhcpv6client:
+      description: >-
+        Configuration for emulated DHCPv6 Client on a single Interface.
+        https://www.rfc-editor.org/rfc/rfc8415.html
+      type: object
+      x-status:
+        status: under_review
+      required: [name]
+      properties:
```

Comment:
In addition, there is some relay server broadcast option for DHCPv4 client. Is that not applicable for DHCPv6 client ?

Reply:
sends a multicast Solicit message to all DHCPv6 servers on the same link

Reply:
DHCPv6 is multicast,DHCPv4 is uss broadcast.